packages/react-scripts/template-hf/views/index.ejs

```diff
@@ -0,0 +1,9 @@
+<%
+  page.secondaryNav = [
+    {"href": appPath("/"), title: 'Stuff', "text": 'Stuff'},
+  ];
+  page.showHelper = true;
```

Comment:
do we fell that every app should have helper and secondaryNav setup by default? what about several other page variables that they could enable?